### PR TITLE
lopper:assists Bug fixes in baremetal_xparameters and baremetal_linker assist files.

### PR DESCRIPTION
**Pull request by onkarharsh**
Few Bug fixes in baremetal_xparameters and baremetal_linker assist files.

### CODE REVIEW

**Comment by onkarharsh** [comment]
Hi @kedareswararao ,
Please review the commits once.

File: lopper/assists/baremetal_xparameters_xlnx.py
Comment (kedareswararao) [inline lopper/assists/baremetal_xparameters_xlnx.py:1]: This change looks fine to me...

File: lopper/assists/baremetallinker_xlnx.py
Comment (kedareswararao) [inline lopper/assists/baremetallinker_xlnx.py:1]: Change looks fine to me.

**Comment by onkarharsh** [comment]
Thanks @kedareswararao .

**Pull request merged**